spec/index.bs

```diff
@@ -6313,6 +6314,12 @@ dictionary GPUComputePipelineDescriptor : GPUPipelineDescriptorBase {
                         </div>
 
                     1. Set |pipeline|.{{GPUPipelineBase/[[layout]]}} to |layout|.
+
+                    1. If any [=uncategorized shader error=] was detected during
+                        [=shader-creation error|shader creation=] of
+                        |descriptor|.{{GPUComputePipelineDescriptor/compute}}.{{GPUProgrammableStage/module}}, or during
+                        the [=pipeline-creation error|pipeline creation=] of |pipeline|:
+                        1. [$generate an operation error$] and make |pipeline| [=invalid=].
```

Comment:
There should also be something in createShaderModule, I think along the lines of "if there's an uncategorized shader error, make the GPUShaderModule invalid without generating any error"